doc/src/Howtorun.tex

```diff
@@ -102,24 +104,116 @@ \section{TEAL for RAVEN}
 </Economics>
 \end{lstlisting}
 
-As one can see, all the specifications of the \textbf{TEAL.CashFlow} module are given in the \xmlNode{Economics} block. The block accepts an attribute called \xmlAttr{verbosity},
+As one can see, all the specifications of the \textbf{TEAL.CashFlow} module are given in the \\\xmlNode{Economics} block. The block accepts an attribute called \xmlAttr{verbosity},
```

Comment:
I don't think these two backslashes are desired, it should just be `\xmlNode{Economics}` I think.

Reply:
This is for a line break since otherwise Economics goes into the margins.